library/src/main/java/com/mapzen/android/graphics/MapzenMap.java

```diff
@@ -179,7 +184,7 @@ public float getZoom() {
    * Set map position without animation.
    */
   public void setPosition(LngLat lngLat) {
-    mapController.setPosition(lngLat);
+    setPosition(lngLat, 0);
```

Comment:
Good eye. I don't believe it's currently guaranteed that an eased call with a duration of zero will produce the exact same result as a non-eased call. We can (and probably should) make this true, but for now I'd recommend not easing with durations of zero.